src/main/java/core/basesyntax/model/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.model;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int base1;
+    private int base2;
+    private int height;
+
+    public IsoscelesTrapezoid(int base1, int base2, int height, String color) {
+        this.base1 = base1;
+        this.base2 = base2;
+        this.height = height;
+        setColor(color);
```

Comment:
```suggestion
        super(color);
```
the same with other figures